datastore/db/queries/inspection/__init__.py

```diff
@@ -382,3 +384,37 @@ def update_inspection(
         raise InspectionUpdateError(f"Invalid input: {str(e)}") from e
     except Exception as e:
         raise InspectionUpdateError(f"Unexpected error: {str(e)}") from e
+
+def get_inspection_factual(cursor, inspection_id):
```

Comment:
This is only a query to suite testing and retrieving data. In the future, retrieving an inspection directly from the OLAP layer enables us to retrieve label_information child object directly with their ids O(1) instead of using the label_id fk which isn't O(1)

Reply:
I understand. If your concern is efficiency, maybe we should just refactor the operational schema. But we can't mix OLTP and OLAP, tempting as it is. One reason for that is that,  in data warehouses, usually OLAPs are not required to be perfectly representative of the real time operational data.

I'm fine with denormalizing the operational schema for efficiency. But in that case, let's not call it OLAP.

Reply:
After discussion, I understand that this function will not be used for operational matters (which is unfortunate if the performance is better). Please don't mind the comment.